src/core/manager.c

```diff
@@ -1867,6 +1867,12 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds, const char *roo
                 /* Third, fire things up! */
                 manager_coldplug(m);
 
+                /* We might have drop-ins for init.scope, but the cgroup is created and realized before
+                 * we load them. Do it again now, after we've got all the units loaded. */
+                Unit *u = manager_get_unit(m, SPECIAL_INIT_SCOPE);
```

Comment:
i think this would be better placed in scope_enumerate_perpetual() where the unit is set up. 

and I think instead of realizing it imemdiately from there, just add it to the cgroup queue for it, i.e, call unit_add_to_cgroup_realize_queue() which you might need to export first (it's currently a static function in cgroup.c).

also, i think slice_make_perpetual() in slice.c needs similar treatment, so that the root slice's properties are updated, too.

Reply:
Moved - but the slice change doesn't seem to be working, for neither the root slice nor system.slice, drop-ins are still ignored. I really need init.scope to work here, so I'll keep this PR about this, and try to figure out the slices separately.